trx/trx_file_memmap.py

```diff
@@ -1736,24 +1736,10 @@ def to_sft(self, resize=False):
     def close(self) -> None:
         """Cleanup on-disk temporary folder and initialize an empty TrxFile"""
         if self._uncompressed_folder_handle is not None:
-            # Problem on Windows that is solving only in Python above 3.10
-            # Step 1: Traverse the directory tree
-            for dirpath, dirnames, filenames in \
-                os.walk(self._uncompressed_folder_handle.name, topdown=False):
-                
-                # Step 2 and 3: Identify and remove symlinks (for files)
-                for filename in filenames:
-                    filepath = os.path.join(dirpath, filename)
-                    if os.path.islink(filepath):
-                        os.unlink(filepath)
-                
-                # Step 2 and 3: Identify and remove symlinks (for directories)
-                for dirname in dirnames:
-                    dir_full_path = os.path.join(dirpath, dirname)
-                    if os.path.islink(dir_full_path):
-                        os.unlink(dir_full_path)
-            
-            # Step 4: Remove the temporary directory
-            shutil.rmtree(self._uncompressed_folder_handle.name)
+            try:
+                self._uncompressed_folder_handle.cleanup()
+            except PermissionError:
+                logging.error("Windows PermissionError, temporary directory {}" +
```

Comment:
Is this going to reraise the error in all cases? I wonder whether you want to implement a custom error and raise that instead, or reraise the original error with the new error message, rather than log.

Reply:
Do we want to code to execute and let the user go delete files manually (or fix permission) or crash?

I was thinking 'Keep going, but warn the user this is taking up disk space'

Reply:
Gotcha. Yes, that makes sense.